JSDemos/Demos/Scheduler/CellTemplates/Angular/app/app.component.html

```diff
@@ -19,27 +19,41 @@
   <div
     *dxTemplate="let dataCell of 'dataCellTemplate'"
     [ngClass]="{
-      'disable-date': isDisableDate(dataCell.startDate),
+      'disable-date':
+        (isHoliday | apply : dataCell.startDate : holidays) ||
+        (isWeekend | apply : dataCell.startDate),
```

Comment:
Let's return method ```isDateDisabled``` to the component.
It's reduce a template code (code with apply pipe + conditions isn't very attractive)

```typescript
// app.component.ts

  isDateDisabled(date: Date) {
    return this.isHoliday(date) || this.isWeekend(date);
  }
```

And here:
```suggestion
      'disable-date': (isDateDisabled | apply : dataCell.startDate)
```